packages/react-components/src/utils/setupConfiguration.ts

```diff
@@ -114,8 +114,30 @@ async function saveConfiguration(
         `${isIngestingMetadataOnly}`
     );
     await saveIndexForIngestion(service, indexName);
+    const isFirstConfiguration = await fetchIsFirstConfiguration();
+    if (isFirstConfiguration) {
+        await updateEventIngestionCronJobInterval(service, 1);
+    }
```

Comment:
I wonder if this is necessary right off the top.

What about defaulting the interval to 1 > complete setup > reload app (a job runs instantly) > check isFirstConfiguration == 1 > update config > reload app

Does this make sense or am I missing something?

Reply:
That'd make sense yes

Reply:
Instead, by default it's scheduled to run once at midnight on january 1st every year until it's configured. At that point it will run once right away and then every minute after the first run is over